winterfell/models/form.js

```diff
@@ -8,7 +8,9 @@ var formSchema = new Schema({
   key: String, // Code name for the form
   responses: Schema.Types.Mixed,
   user: Schema.Types.ObjectId,
-  claim: Schema.Types.ObjectId
+  claim: Schema.Types.ObjectId,
+  answered: Number,
+  answerable: Number
```

Comment:
ok I see. I think that works pretty well for now. I think it's okay to have some things already considered "complete" even if it's blank.